src/components/help/contact-support-modal.tsx

```diff
@@ -22,26 +22,109 @@ import {
 import { ConnectWallet } from "@thirdweb-dev/react";
 import { useLoggedInUser } from "@3rdweb-sdk/react/hooks/useLoggedInUser";
 
-const productOptions = [
-  "Connect",
-  "Contracts",
-  "Payments",
-  "Infrastructure",
-  "Account",
-  "Billing",
-  "Other",
+const defaultDescription =
+  "Please describe the issue you're encountering in detail, including steps that led to the error, any error messages, troubleshooting steps you've already taken, and the product(s), dashboard, or SDKs involved.";
+
+type ProductOption = {
+  label: string;
+  problemAreas: Array<{
+    label: string;
+    affectedAreas: Array<{ label: string; customDescriptionPrompt?: string }>;
+  }>;
+};
+
+const productOptions: ProductOption[] = [
+  {
+    label: "Connect",
+    problemAreas: [
+      {
+        label: "Embedded wallet login issues",
+        affectedAreas: [
+          { label: "Dashboard", customDescriptionPrompt: defaultDescription },
+          { label: "Application", customDescriptionPrompt: defaultDescription },
+        ],
+      },
+      {
+        label: "Embedded wallet transaction issues",
+        affectedAreas: [],
+      },
+      {
+        label: "Embedded wallet custom Auth",
+        affectedAreas: [],
+      },
+      {
+        label: "Account Abstraction",
+        affectedAreas: [],
+      },
+      {
+        label: "React SDK",
+        affectedAreas: [],
+      },
+      {
+        label: "TypeScript SDK",
+        affectedAreas: [],
+      },
+    ],
+  },
+  {
+    label: "Contracts",
+    problemAreas: [],
+  },
+  {
+    label: "Payments",
+    problemAreas: [],
+  },
+  {
+    label: "Infrastructure",
+    problemAreas: [],
+  },
+  {
+    label: "Account",
+    problemAreas: [],
+  },
+  {
+    label: "Billing",
+    problemAreas: [],
+  },
+  {
+    label: "Others",
+    problemAreas: [],
+  },
 ];
 
-export const ContactSupportModal = () => {
+export default function ContactSupportModal() {
```

Comment:
generally prefer named exports over default exports -> I know this is required for the dynamic import (see separate question about that)